.github/workflows/js-tests.yml

```diff
@@ -5,7 +5,29 @@ on:
     branches:
       - develop
       - main
+    # Only run if CSS/JS/MD-related files changed.
+    paths:
+      - '.github/workflows/js-tests.yml'
+      - '**.js'
+      - '**.json'
```

Comment:
Any json file would also include `composer.json` so we should be a tiny bit more specific, or at least exclude a few specific ones like this `.eslintrc.json`, and `feature-flags.json`.